pkg/common/alluxio.go

```diff
@@ -12,13 +12,15 @@ const (
 
 	ALLUXIO_DATA_LOADER_IMAGE_ENV = "AlluxioDataLoaderImage"
 
-	// DEFAULT_ALLUXIO_DATA_LOADER_IMAGE = "registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio-data-loader:v2.2.0"
-
 	ALLUXIO_INIT_IMAGE_ENV = "ALLUXIO_INIT_IMAGE_ENV"
 
 	ALLUXIO_RUNTIME_IMAGE_ENV = "ALLUXIO_RUNTIME_IMAGE_ENV"
 
 	ALLUXIO_FUSE_IMAGE_ENV = "ALLUXIO_FUSE_IMAGE_ENV"
 
 	DEFAULT_ALLUXIO_INIT_IMAGE = "registry.cn-hangzhou.aliyuncs.com/fluid/init-users:v0.3.0-1467caa"
```

Comment:
I think this is not just "DEFAULT_ALLUXIO_INIT_IMAGE" now, how about "DEFAULT_INIT_IMAGE"

Reply:
I agree.